bot/exts/moderation/voice_gate.py

```diff
@@ -28,12 +29,26 @@
     "activity_blocks": f"have been active for fewer than {GateConf.minimum_activity_blocks} ten-minute blocks",
 }
 
+VOICE_PING = (
+    "Hello, {0}! Wondering why you can't talk in the voice channels? "
+    "Use the `!voiceverify` command in here to verify. "
+    "If you don't yet qualify, you'll be told why!"
+)
+
 
 class VoiceGate(Cog):
     """Voice channels verification management."""
 
-    def __init__(self, bot: Bot):
+    # RedisCache[t.Union[discord.User.id, discord.Member.id], t.Union[discord.Message.id, bool]]
+    redis_cache = RedisCache()
+
+    def __init__(self, bot: Bot) -> None:
         self.bot = bot
+        self._init_task = self.bot.loop.create_task(self._async_init())
+
+    async def _async_init(self) -> None:
+        await self.bot.wait_until_guild_available()
+        self._voice_verification_channel = self.bot.get_channel(Channels.voice_gate)
```

Comment:
I wonder if this is completely safe, i.e. could the instance get stale on bot restart or similar? We do it elsewhere too, just wondering. I'm guessing the linter doesn't yell about the lack of docstring because it's a private func.

Reply:
Ended up removing the `_async_init` in favor of moving it all to the `on_voice_state_update` event listener.